src/config/graphical-session/graphics-drivers/nvidia.md

```diff
@@ -0,0 +1,79 @@
+# NVIDIA
+
+## nouveau (Open Source Driver)
+
+Install the Mesa DRI driver `mesa-dri` and optionally the Xorg driver
+`xf86-video-nouveau`.
+
+> Note: This is already included in the `xorg` meta-package, but is needed when
+> installing via `xorg-minimal`.
```

Comment:
No blockquote here, please. 

What is "this" referring to? `mesa-dri`, `xf86-video-nouveau`, both?

Reply:
"this" is referring to `xf86-video-nouveau`. I will clarify.